src/annotated_lcf.fun

```diff
@@ -10,6 +16,11 @@ struct
      goal : goal,
      metadata : metadata}
 
+  (* If the tactic succeeds annotating it will do
+   * nothing. The tactic is only effected by operations
+   * if it throws an exception in which case the exception
+   * is elaborated into a RefinementFailed and reraised.
+   *)
```

Comment:
- add comma after succeeds
- change last sentence to say something like "The tactic is only **a**ffected in case it throws an exception, which is then wrapped in RefinementFailed an reraised."